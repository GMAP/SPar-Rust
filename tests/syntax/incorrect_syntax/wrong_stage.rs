extern crate spar_rust;
use spar_rust::to_stream;

pub fn main() {
    let a = 1;
    to_stream!(INPUT(a: u32), INPUT(a: u32), {
        STAGE();
    });
}
