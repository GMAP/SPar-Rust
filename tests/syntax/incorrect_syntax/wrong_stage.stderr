error: multiple INPUTs aren't allowed
 --> tests/syntax/incorrect_syntax/wrong_stage.rs:6:31
  |
6 |     to_stream!(INPUT(a: u32), INPUT(a: u32), {
  |                               ^^^^^
